server/routes/dashboard.js

```diff
@@ -0,0 +1 @@
+
```

Comment:
Should this empty file be deleted?

Reply:
sorry I was planning to create a route there should have deleted before pull request.Thank you.